CHANGELOG.md

```diff
@@ -1,47 +1,64 @@
-# 2.1.0
+# 2.0.0
+
+<!-- - Bugfix | Change | Feature | Documentation | Security -->
+
+## Data migration notes
 
-This release adds a logging collection to the database.
+In version 2.0.0, the ``project`` collection is partly filled by harvesting from
+external data source, and partly filled by means of users making edits in the
+admin interface. This means that version 2.0.0 of the Research Software
+Directory requires changes to the database. Below are the steps to migrate data
+from 1.2.0 to 2.0.0. Furthermore, the frontend now shows information for page
+maintainers, for which a new MongoDB collection ``logging`` is needed.
 
 ```
-docker-compose exec database mongo rsd
-db.createCollection("logging")
+$ docker-compose exec database mongo rsd
 ```
 
-- Added logs of harversters to frontend software item page
+**Create** collection "logging":
 
-# 2.0.0
+```
+db.createCollection("logging")
+```
 
-In version 2.0.0, the ``project`` collection is partly filled by harvesting from external data source, and partly filled by means of users making edits in the admin interface. This means that version 2.0.0 of the Research Software Directory requires changes to the database. Below are the steps to migrate data from 1.2.0 to 2.0.0.
+**Remove** all ``release`` documents entirely:
 
 ```
-$ docker-compose exec database mongo rsd
+db.release.deleteMany({})
 ```
 
-**Add** fields ``output`` and ``impact`` to all ``project`` documents:
+**Remove** all ``project`` and ``project_cache`` documents  entirely:
 
 ```
-db.project.update({}, {$set: {"output": []}}, {"multi": true})
-db.project.update({}, {$set: {"impact": []}}, {"multi": true})
+db.project.deleteMany({})
+db.project_cache.deleteMany({})
 ```
 
-**Rename** field ``url`` to ``corporateUrl`` for all ``project`` documents:
+In another terminal, run the project harvester:
 
 ```
-db.project.update({}, {$rename: {"url": "corporateUrl"}}, {"multi": true})
+$ source rsd-secrets.env
+$ docker-compose exec harvesting python app.py harvest projects
 ```
 
-**Empty** the value of ``principalInvestigator`` for all ``project`` documents:
+Back in the Mongo terminal, **Add** fields ``output`` and ``impact`` to all
+``project`` documents:
 
 ```
-db.project.update({}, {$set: {"principalInvestigator": ""}}, {"multi": true})
+db.project.update({}, {$set: {"output": []}}, {"multi": true})
+db.project.update({}, {$set: {"impact": []}}, {"multi": true})
 ```
 
-**Remove** all ``release`` documents entirely:
+Then, update the project identifiers as used in the ``software`` collection by
+copy-pasting the contents of
+[data-migration-1.x-to-2.js](/data-migration-1.x-to-2.js) into the Mongo shell.
```

Comment:
This did not work for me, the mongo shell is started and closed again, but it doesnt seem to run the script. To avoid making things more complicated than they need to be I decided against docker copying the migration script into the container, and just went back to copy-pasting.

Reply:
I think the `-T` should have been removed from the command. Command needs tty while `-T` disables tty. 

Copy/pasting is fine aswell.